test/test_simulation_sequence.jl

```diff
@@ -0,0 +1,63 @@
+@testset "Simulation Sequence" begin
+    feedforward_chronologies = Dict(
+        ("UC" => "HAUC") => Synchronize(periods = 24),
+        ("HAUC" => "ED") => RecedingHorizon(),
+        ("ED" => "AGC") => RecedingHorizon(),
+    )
+    ini_cond_chronology = InterStage()
+    order = Dict(1 => "DAUC", 2 => "HAUC", 3 => "ED", 4 => "AGC")
+    intervals =
+        Dict("DAUC" => Hour(24), "HAUC" => Hour(1), "ED" => Minute(5), "AGC" => Minute(1))
+    horizons = Dict("DAUC" => 48, "HAUC" => 24, "ED" => 12, "AGC" => 6)
+
+    test_sequence = SimulationSequence(
+        order = order,
+        feedforward_chronologies = feedforward_chronologies,
+        step_resolution = Hour(24),
+        horizons = horizons,
+        intervals = intervals,
+        ini_cond_chronology = ini_cond_chronology,
+    )
+
+    @test length(findall(x -> x == 4, test_sequence.execution_order)) == 24 * 60
```

Comment:
Now that execution_order is a vector of stage numbers, do we still need stage numbers? i.e. could we instead just use the stage names and keep a vector of those? That could simplify the interface.

Reply:
I prefer to keep it to streamline the construction and running of the stages. Also, this will force the users to explicitly state the order they expect instead of having to infer it from the order provided. We can come back to this once the whole process is finished.